pkg/terraform/gather/ovirt/ip.go

```diff
@@ -55,9 +55,16 @@ func getReportedDevices(c *ovirtsdk4.Connection, vmID string) (*ovirtsdk4.Report
 	return reportedDeviceSlice, nil
 }
 
-func findVirtualMachineIP(c *ovirtsdk4.Connection, moRefValue string) (string, error) {
+func findVirtualMachineIP(moRefValue string) (string, error) {
 
-	reportedDeviceSlice, err := getReportedDevices(c, moRefValue)
+	client, err := ovirt.NewConnection()
```

Comment:
Why are we creating a new oVirt connection here?

Reply:
moved to the connection one level up (outside the for loop)